lib/cocoapods_plugin.rb

```diff
@@ -0,0 +1,19 @@
+require 'cocoapods'
+
+Pod::Installer
+  .prepend(Module.new do
+             def perform_post_install_actions
+               super
+
+               return unless plugins.key?('refinement')
```

Comment:
How much will this plugin slow down local bepi? Should we avoid running it in local development?

Reply:
It's up to a user if they don't want to always run with the plugin enabled.
For what it's worth, it took less than half a second to run this entire hook on a `pod install` that had 383 pods.